go/common/crypto/signature/signers/ledger/ledger_signer.go

```diff
@@ -0,0 +1,85 @@
+package ledger
+
+import (
+	"encoding/binary"
+
+	"github.com/ethereum/go-ethereum/accounts"
```

Comment:
Why do we need to use anything from ethereum? Is the only reason for importing this the `DefaultRootDerivationPath`? Can't we use our own key derivation path? If we really need to use that, just copy the value here instead of importing the huge pile of geth.